docs/manual/features.rst

```diff
@@ -0,0 +1,106 @@
+.. title:: Features
+
+.. _features:
+
+########
+Features
+########
+
+REST API for user object operations
+===================================
+
+At its core, Thunder is a REST API that provides endpoints to manage user accounts and information.
+Your frontend application can use Thunder to create, retrieve, update, and delete user accounts.
+All of the user information is stored in a database that Thunder interfaces with.
+
+.. note::
+
+    Thunder currently only has support for DynamoDB databases, so an AWS account is required to run
```

Comment:
This sounds weird to me. It sounds better to me if it was 'Currently Thunder only supports DynamoDB databases, so...' but maybe you can play with it.